.Gemfile-rest-client-1.8.rb

```diff
@@ -0,0 +1,11 @@
+# Allow travis to test rest-client 1.x.
```

Comment:
Just to double-check this works: 
https://travis-ci.org/abonas/kubeclient/jobs/226372922 run with `Using rest-client 1.8.0`
(using .Gemfile-rest-client-1.8.rb)
https://travis-ci.org/abonas/kubeclient/jobs/226372920 run with `Using rest-client 2.0.2`
(using Gemfile uses gemspec, which doesn't specify rest-client version, so we get latest)